src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml

```diff
@@ -41,6 +41,14 @@
                 <Setter Property="FontSize" Value="12"/>
                 <Setter Property="TextAlignment" Value="Left"/>
                 <Setter Property="PagePadding" Value="0"/>
+
+                <Style.Resources>
+                    <Style TargetType="Image">
+                        <Setter Property="Stretch" Value="UniformToFill" />
+                        <Setter Property="StretchDirection" Value="DownOnly" />
+                        <Setter Property="MaxWidth" Value="{Binding ActualWidth, ElementName=bodyMarkdown}" />
```

Comment:
Is there a better way to bind this value? Coupling it specifically to `ElementName` doesn't seem very smart or reusable. 🔬 🐶